nano/node/node.cpp

```diff
@@ -253,6 +277,7 @@ void nano::network::flood_message (nano::message const & message_a)
 	auto list (node.network.udp_channels.list_fanout ());
```

Comment:
This is where we should be additionally searching for channels in tcp_channels that are not set to bootstrap_only. In the future this will additionally search for channels in channel_quic or whatever.

Reply:
Updated